lib/KTable/index.vue

```diff
@@ -0,0 +1,480 @@
+      sortable: {
+        type: Boolean,
+        default: true,
+      },
+    },
+    data() {
+      return {
+        focusedRowIndex: null,
+        focusedColIndex: null,
+        hoveredRowIndex: null,
+      };
+    },
+    computed: {
+      coreOutlineFocus() {
+        return {
+          ':focus': {
+            ...this.$coreOutline,
+            outlineOffset: '-2px',
+          },
+        };
+      },
+    },
+    methods: {
+      handleKeydown(event, rowIndex, colIndex) {
+        const key = event.key;
+        const totalRows = this.rows.length;
+        const totalCols = this.headers.length;
+
+        let nextRowIndex = rowIndex;
+        let nextColIndex = colIndex;
+        // Identify all focusable elements inside the current cell
+        const currentCell = this.getCell(rowIndex, colIndex);
+        const focusableElements = currentCell
+          ? Array.from(
+              currentCell.querySelectorAll(
+                'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+              )
+            )
+          : [];
+        const focusedElementIndex = focusableElements.indexOf(document.activeElement);
+        switch (key) {
+          case 'ArrowUp':
+            if (rowIndex === -1) {
+              nextRowIndex = totalRows - 1;
+            } else {
+              nextRowIndex = rowIndex - 1;
+            }
+            break;
+          case 'ArrowDown':
+            if (rowIndex === -1) {
+              nextRowIndex = 0;
+            } else if (rowIndex === totalRows - 1) {
+              nextRowIndex = -1;
+            } else {
+              nextRowIndex = (rowIndex + 1) % totalRows;
+            }
+            break;
+          case 'ArrowLeft':
+            if (rowIndex === -1) {
+              if (colIndex > 0) {
+                nextColIndex = colIndex - 1;
+              } else {
+                nextColIndex = totalCols - 1;
+                nextRowIndex = totalRows - 1;
+              }
+            } else if (colIndex > 0) {
+              nextColIndex = colIndex - 1;
+            } else {
+              nextColIndex = totalCols - 1;
+              nextRowIndex = rowIndex > 0 ? rowIndex - 1 : -1;
+            }
+            break;
+          case 'ArrowRight':
+            if (colIndex === totalCols - 1) {
+              if (rowIndex === totalRows - 1) {
+                nextColIndex = 0;
+                nextRowIndex = -1;
+              } else {
+                nextColIndex = 0;
+                nextRowIndex = rowIndex + 1;
+              }
+            } else {
+              nextColIndex = colIndex + 1;
+            }
+            break;
+          case 'Enter':
+            if (rowIndex === -1 && this.sortable) {
+              this.handleSort(colIndex);
+            }
+            break;
+          case 'Tab':
+            if (focusableElements.length > 0) {
+              if (!event.shiftKey) {
+                if (focusedElementIndex < focusableElements.length - 1) {
```

Comment:
```suggestion
                // if navigating between more focusable elements within the cell
                if (focusedElementIndex < focusableElements.length - 1) {
```

Just a comment suggestion - it became clear, but took me some time to get it because at first I didn't realize there could be more focusable elements.